### PR TITLE
utils_misc: add get_linux_drive_path function

### PR DESCRIPTION
**Pull request by luckyh**
This function provides a method to get target drive path in guest by serial or wwn.

ID: 1279721 1279719 1272397

### CODE REVIEW

File: virttest/utils_misc.py [hunk 1]
```diff
@@ -2749,6 +2749,29 @@ def format_guest_disk(session, did, mountpoint, size, fstype, ostype):
...
+def get_linux_drive_path(session, did):
+    """
+    Get drive path in guest by drive serial or wwn
+
+    :param session: session object to guest.
+    :param did: drive serial or wwn.
+    :return String: drive path
+    """
+    cmd = 'for dev_path in `ls -d /sys/block/*`; do '
+    cmd += 'echo `udevadm info -q property -p $dev_path`; done'
```
Comment (xutian) [inline virttest/utils_misc.py]: is default timeout  enough for multi-disk test ?

Comment (luckyh) [inline virttest/utils_misc.py]: Seems so.

multi_disk.max_disk(~25 disks):
```
18:48:49 client           L1042 DEBUG| Sending command: for dev_path in `ls -d /sys/block/*`; do echo `udevadm info -q property -p $dev_path`; done
18:48:50 client           L1042 DEBUG| Sending command: echo $?
```

**Comment by xutian** [comment]
better to pass a timeout args and set default values for that args. if default timeout is long enough, we can set it when we call it.

**Comment by luckyh** [comment]
@xutian have modified the function, would you please help review it, thanks!

**Comment by spcui** [comment]
LGTM, Acked-by: scui@redhat.com

**Comment by xutian** [comment]
Merged, thanks @luckyh  and @spcui .

**Pull request merged**